src/test/java/com/lukaspradel/steamapi/webapi/client/SteamWebApiClientTest.java

```diff
@@ -850,4 +852,40 @@ public void testProcessResolveVanityUrlNoMatch() throws IOException, SteamApiExc
 		assertEquals(response.getSuccess(), 42);
 		assertEquals(response.getMessage(), "No match");
 	}
+
+	@Test
+	public void testProcessGetPlayerItemsRequest() throws SteamApiException, IOException {
+
+		String steamId = "76561197960435530";
+
+		GetPlayerItemsRequest getPlayerItemsRequest = SteamWebApiRequestFactory
+				.createGetPlayerItemsRequest(steamId);
+
+		String mockAnswer = readResourceAsString("tf2/GetPlayerItems.json");
+
+		when(requestHandlerMock.getWebApiResponse(getPlayerItemsRequest))
+				.thenReturn(mockAnswer);
+
+		GetPlayerItems getPlayerItems = client
+				.<GetPlayerItems> processRequest(getPlayerItemsRequest);
+
+		assertNotNull(getPlayerItems);
+		assertNotNull(getPlayerItems.getResult());
+		assertEquals(getPlayerItems.getResult().getStatus(), 1);
+		assertEquals(getPlayerItems.getResult().getNumBackpackSlots(), 1700);
+
+		// Get Robin Walker's 'Valve' quality items
+		var valveQualityItems = getPlayerItems.getResult().getItems()
+												.stream()
+												.filter(item -> item.getQuality().equals(8D))
+												.toList();

```

Comment:
For now, this library is on Java 11 hence `toList()` is not available yet and results in compile error (see build workflow)

```suggestion
												.collect(Collectors.toList());

```